.circleci/config.yml

```diff
@@ -89,7 +69,26 @@ jobs:
       - setup_remote_docker:
           docker_layer_caching: true
       - run:
-          lein test
+          name: Start External Services (docker-compose up)
+          command: docker-compose -f docker-compose.yml up -d
+      - run:
+          name: Run Tests
+          command: |
+            docker create \
+              -v /home/circleci/.m2 \
+              -v /home/circleci/jackdaw \
+              --name data circleci/clojure:lein-2.8.1 \
+              /bin/true
+
+            docker cp /home/circleci/.m2 data:/home/circleci/
+            docker cp /home/circleci/jackdaw data:/home/circleci/
+
+            docker run \
+              --env-file .env.circle \
+              --workdir /home/circleci/jackdaw \
+              --volumes-from data \
+              --network jackdaw_default \
+              circleci/clojure:lein-2.8.1 lein test
```

Comment:
The CircleCI documentation recommends the approach used in master. Please see ["Using Multiple Docker Images"](https://circleci.com/docs/2.0/executor-types/#using-multiple-docker-images). I suggest we stay closer to that. 

https://github.com/FundingCircle/jackdaw/pull/66 shows how we can extend this to test against multiple versions of the Confluent Platform: https://github.com/FundingCircle/jackdaw/pull/66. The tests run in parallel.